vendor/bat-native-ledger/src/bat/ledger/internal/uphold/uphold_util.cc

```diff
@@ -187,9 +187,11 @@ bool one_of(T&& t, Ts&&... ts) {
   return match;
 }
 
-void LogWalletStatusChange(LedgerImpl* ledger,
-                           absl::optional<type::WalletStatus> from,
-                           type::WalletStatus to) {
+bool show_wallet_new_verified = true;
```

Comment:
Maybe rename to `g_show_newly_verified_wallet` and stick it in an anonymous namespace (see comment below).